pinot-spi/src/main/java/org/apache/pinot/spi/utils/builder/ControllerRequestURLBuilder.java

```diff
@@ -344,6 +344,10 @@ public String forSegmentMetadata(String tableName, String segmentName) {
     return StringUtil.join("/", _baseUrl, "segments", tableName, encode(segmentName), "metadata");
   }
 
+  public String forSegmentLineage(String tableName, String tableType) {
```

Comment:
`forListAllSegmentLineages` would be better naming (following the convention with other functions)